collect_app/src/androidTest/java/org/odk/collect/android/feature/smoke/AllWidgetsFormTest.java

```diff
@@ -295,6 +296,9 @@ public void testImageWithoutChooseWidget() {
     public void testSelfieWidget() {
         Screengrab.screenshot("selfie-widget");
 
+        onView(withText("Take Picture")).perform(click());
```

Comment:
I found it useful to have this test make sure the selfie screen can open without any problems during development. It'd probably be nice to add this to other widgets if we feel like we need it when working on them.